mods/ra/maps/soviet-soldier-volkov-n-chitzkoi/rules.yaml

```diff
@@ -0,0 +1,233 @@
+2TNK:
+	Buildable:
+		Prerequisites: ~vehicles.allies, ~techlevel.medium
+        
+CA:
+	Buildable:
+		Prerequisites: ~syrd, ~techlevel.high
+        
+C2:
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
+	Armament:
+		Weapon: Pistol
+	AttackFrontal:
+	WithInfantryBody:
+		DefaultAttackSequence: shoot
+        
+C3:
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
+	Armament:
+		Weapon: Pistol
+	AttackFrontal:
+	WithInfantryBody:
+		DefaultAttackSequence: shoot
+        
+C4:
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
+	Armament:
+		Weapon: Pistol
+	AttackFrontal:
+	WithInfantryBody:
+		DefaultAttackSequence: shoot
+        
+C5:
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
+	Armament:
+		Weapon: Pistol
+	AttackFrontal:
+	WithInfantryBody:
+		DefaultAttackSequence: shoot
+        
+C6:
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
+	Armament:
+		Weapon: Pistol
+	AttackFrontal:
+	WithInfantryBody:
+		DefaultAttackSequence: shoot
+        
+C8:
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
+	Armament:
+		Weapon: Pistol
+	AttackFrontal:
+	WithInfantryBody:
+		DefaultAttackSequence: shoot
+        
+C9:
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
+	Armament:
+		Weapon: Pistol
+	AttackFrontal:
+	WithInfantryBody:
+		DefaultAttackSequence: shoot
+        
+C10:
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
+	Armament:
+		Weapon: Pistol
+	AttackFrontal:
+	WithInfantryBody:
+		DefaultAttackSequence: shoot
+        
+V01:
+	SpawnActorOnDeath:
+		Actor: healcrate
+        
+HOSP:
+	SpawnActorOnDeath:
+		Actor: healcrate
+        
+MONEYCRATE:
+	GiveCashCrateAction:
+		Amount: 2000
+
+^Vehicle:
+	-Demolishable:
+
+AFAC:
+	Inherits: BIO
+	Buildable:
+		Prerequisites: ~disabled
+	Health:
+		HP: 400000
+	Tooltip:
+		Name: Alloy Facility
+	-ExternalCapturable:
+	-ExternalCapturableBar:
+	-EngineerRepairable:
+	-GpsDot:
+	RenderSprites:
+		Image: BIO
+
+CCEN:
+	Inherits: ATEK
+	Buildable:
+		Prerequisites: ~disabled
+	Tooltip:
+		Name: Control Center
+	-ExternalCapturable:
+	-ExternalCapturableBar:
+	-EngineerRepairable:
+	GpsPower:
+		Prerequisites: ~disabled
+	Power:
+		Amount: 200
+	RenderSprites:
+		Image: ATEK
+
+HGUN:
+	Inherits: GUN
+	Buildable:
+		Prerequisites: ~disabled
+	Valued:
+		Cost: 2800
+	Tooltip:
+		Name: Heavy Turret Prototype
+	Health:
+		HP: 350000
+	Armor:
+		Type: Heavy
+	RevealsShroud:
+		Range: 8c0
+	Armament:
+		Weapon: HeavyTurretGun
+	RenderSprites:
+		Image: GUN
+
+ZKOI:
+	Inherits: DOG
+	Buildable:
+		Prerequisites: ~disabled
+	Valued:
+		Cost: 2000
+	Tooltip:
+		Name: Chitzkoi
+		GenericName: Cyborg Dog
+	Health:
+		HP: 100000
+	-Crushable:
+	RevealsShroud:
+		Range: 7c0
+	Armament:
+		Weapon: CydogJaw
+	RenderSprites:
+		Image: DOG
+
+VOLK:
+	Inherits: GNRL
+	Buildable:
+		Prerequisites: ~disabled
+	Valued:
+		Cost: 3000
+	Tooltip:
+		Name: Volkov
+	Health:
+		HP: 250000
+	Armor:
+		Type: Heavy
+	-Crushable:
+	RevealsShroud:
+		Range: 10c0
+	Demolition:
+	Mobile:
+		-Voice: Move
```

Comment:
Why do these voices need to be removed?

Reply:
Because these voices can't be used with the GenericVoice voiceset. It make the game crash.